gfx/portraits/portrait_modifiers/01_hairstyles.txt

```diff
@@ -1376,4 +1376,91 @@ hairstyles = {
 			fixed_appearance_modifiers = yes
 		}
 	}
+	#  Warcraft 
+	dwarven_straight = {
+        dna_modifiers = {
+            accessory = {
+                mode = add
+                gene = hairstyles
+                template = western_hairstyles_straight
+                range = { 0 1 } # For the randomness to work correctly
+            }
+        }   
+        weight = {
+            base = 0
+            modifier = {
+                add = default_hair_chance
+				OR = {
+					has_gene = {
+						category = gene_hair_type
+						template = hair_straight
+					}
+					has_gene = {
+						category = gene_hair_type
+						template = hair_straight_thin_beard
+					}
+				}
+
+				# Warcraft
+                portrait_dwarven_hairstyles_trigger = yes
+            }
+            
+			# Warcraft
+			fixed_appearance_modifiers = yes
+        }
+    }
+
+    dwarven_wavy = {
+        dna_modifiers = {
+            accessory = {
+                mode = add
+                gene = hairstyles
+                template = western_hairstyles_wavy
+                range = { 0 1 } # For the randomness to work correctly
+            }
+        }   
+        weight = {
+            base = 0
+            modifier = {
+                add = default_hair_chance
+                has_gene = {
+                    category = gene_hair_type
+                    template = hair_wavy
+                }
+
+				# Warcraft
+                portrait_dwarven_hairstyles_trigger = yes
+            }
+            
+			# Warcraft
+			fixed_appearance_modifiers = yes
+        }
+    }
+
+    dwarven_curly = {
+        dna_modifiers = {
+            accessory = {
+                mode = add
+                gene = hairstyles
+                template = western_hairstyles_curly
+                range = { 0 1 } # For the randomness to work correctly
+            }
+        }   
+        weight = {
+            base = 0
+            modifier = {
+                add = default_hair_chance
+                has_gene = {
+                    category = gene_hair_type
+                    template = hair_curly
+                }
+
+				# Warcraft
+                portrait_dwarven_hairstyles_trigger = yes
+            }
+           
+			# Warcraft
+			fixed_appearance_modifiers = yes
+        }
+    }
```

Comment:
What is the point if you refer to already referred `western_hairstyles_straight`, etc. templates?